packages/cli/src/cmds/record/state/testCasesComplete.ts

```diff
@@ -9,7 +9,8 @@ export default async function testCasesComplete(recordContext: RecordContext): P
   // Handle command failures here, rather than in a separate state, so we maintain compatibility
   // with the Azure Function that processes telemetry events.
   if (recordContext.failures > 0 || recordContext.appMapsCreated === 0) {
-    UI.warn(`\n${chalk.yellow('!')} The test commands failed to create AppMaps\n`);
+    if (recordContext.appMapsCreated === 0)
```

Comment:
Isn’t the main issue that this command can think that no AppMaps were created when the test command runs for a second time and updates a bunch of existing AppMaps?

Reply:
If AppMap time stamps are updated then this command can assume that AppMaps were in fact created and that would be a good fix to make as well.

Reply:
That's this issue: https://github.com/getappmap/appmap-js/issues/719 . I thought, for the sake of small batches, it was reasonable to split them up.